<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1111</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(144, 144, 144);</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>9</y>
       <width>941</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #457b9d;
border-radius: 10px;



border: 1px solid black; /* Sets a black border with a width of 2 pixels */
</string>
     </property>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>50</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icon/window-split-horizontal-128.gif</normaloff>../icon/window-split-horizontal-128.gif</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Outlook</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icon/user.gif</normaloff>../icon/user.gif</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icon/search.gif</normaloff>../icon/search.gif</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff;
border-radius: 10px; /* Sets a border radius of 10 pixels */
border: 2px solid black; /* Sets a black border with a width of 2 pixels */
</string>
      </property>
      <property name="placeholderText">
       <string>Search....</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClose">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icon/exist.gif</normaloff>../icon/exist.gif</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>171</width>
       <height>651</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#a8dadc;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnHome">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(69, 123, 157);</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icon/home.gif</normaloff>../icon/home.gif</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMaps">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(69, 123, 157);</string>
      </property>
      <property name="text">
       <string>Map</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icon/location.gif</normaloff>../icon/location.gif</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDOrders">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(69, 123, 157);</string>
      </property>
      <property name="text">
       <string>Delivered Orders</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icon/delivered.gif</normaloff>../icon/delivered.gif</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCustomers">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(69, 123, 157);</string>
      </property>
      <property name="text">
       <string>Customers</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icon/customers.gif</normaloff>../icon/customers.gif</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAOrders">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #457b9d;</string>
      </property>
      <property name="text">
       <string>Active Orders</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icon/incoming-order.gif</normaloff>../icon/incoming-order.gif</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>610</y>
        <width>151</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #e63946;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icon/icon/exist.gif</normaloff>:/icon/icon/exist.gif</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 100px solid black;
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../medicine.jpg</normaloff>../../medicine.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>300</width>
        <height>200</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMedicines">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(69, 123, 157);</string>
      </property>
      <property name="text">
       <string>Medicines</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icon/side-bar.gif</normaloff>../icon/side-bar.gif</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="mainBody">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>941</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#ffc8dd;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>901</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Schoolbook</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#a8dadc;
border: 1px solid black;</string>
      </property>
      <property name="text">
       <string>Pharma Distribution System</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGraph">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>540</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #ff9999;
    color: black;
}

QPushButton:hover {
    background-color: #ffcccc; 
}

QPushButton:pressed {
    background-color: #ff9999;
}
</string>
      </property>
      <property name="text">
       <string>Sketch Graph</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMap">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>540</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: #a2d2ff;
    color: black;
}

QPushButton:hover {
	background-color: #bde0fe;
}

QPushButton:pressed {
	background-color: #a2d2ff;
}</string>
      </property>
      <property name="text">
       <string>View Map</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSearch">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Search Location in Graph...</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioBFS">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>BFS</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioDFS">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>160</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DFS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearch">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #66cc66; 
    color: black;
}

QPushButton:hover {
    background-color: #ccffcc; 
}

QPushButton:pressed {
    background-color: #99ff99;
}
</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnInsertNode">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>150</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #bb2020; 
    color: white; 
}

QPushButton:hover {
    background-color: #cc0000; 
}

QPushButton:pressed {
    background-color: #990000;
}
</string>
      </property>
      <property name="text">
       <string>Insert Node</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Find the Minimum Cost between two Nodes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>350</y>
        <width>811</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Route between the Nodes...</string>
      </property>
      <property name="textFormat">
       <enum>Qt::MarkdownText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtStartingNode">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Starting Node...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtEndingNode">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>290</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>Ending Node...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFind">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>290</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #66cccc; /* Tealish shade */
    color: black;
}

QPushButton:hover {
    background-color: #99dddd; /* Lighter shade on hover */
}

QPushButton:pressed {
    background-color: #66cccc; /* Same as normal state when pressed */
}
</string>
      </property>
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtOutput">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>290</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid black;
border-radius: 10px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Output...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>591</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enjoy the best version of program!!!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
